assets/templates/cards-component.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>UI library Doppler - Cards Component</title>
+    <meta name="keywords" content="UI library Doppler - Layout Reports" />
+    <meta name="description" content="First layout of Reports" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="canonical" href="" />
+    <!-- Open Graph metatags -->
+    <meta property="og:title" content="Layout Reports" />
+    <meta property="og:type" content="article" />
+    <meta property="og:image" content="Layout Reports" />
+    <meta property="og:url" content="Layout Reports" />
+    <meta property="og:description" content="Layout Reports" />
+    <meta property="og:site_name" content="Layout Reports" />
+    <!-- Stylesheets and polyfills -->
+    <link rel="stylesheet" type="text/css" href="./css/styles.css" />
+    <!-- Favicons -->
+    <link
+      rel="apple-touch-icon-precomposed"
+      sizes="144x144"
+      href="https://www.fromdoppler.com/apple-touch-icon-144x144-precomposed.png"
+    />
+    <link
+      rel="apple-touch-icon-precomposed"
+      sizes="114x114"
+      href="https://www.fromdoppler.com/apple-touch-icon-114x114-precomposed.png"
+    />
+    <link
+      rel="apple-touch-icon-precomposed"
+      sizes="72x72"
+      href="https://www.fromdoppler.com/apple-touch-icon-72x72-precomposed.png"
+    />
+    <link
+      rel="apple-touch-icon-precomposed"
+      sizes="57x57"
+      href="https://www.fromdoppler.com/apple-touch-icon-57x57-precomposed.png"
+    />
+    <link
+      rel="apple-touch-icon-precomposed"
+      href="https://www.fromdoppler.com/apple-touch-icon-precomposed.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      href="https://www.fromdoppler.com/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="https://www.fromdoppler.com/favicon.ico"
+    />
+    <link
+      rel="shortcut icon"
+      type="image/x-icon"
+      href="https://www.fromdoppler.com/favicon.ico"
+    />
+  </head>
+
+  <body data-active-menu="28" class="dp-library">
+    <div class="dp-app-container">
+      <header class="m-t-30 m-b-30 m-l-30" style="display: flex;">
+        <a
+          href="index.html"
+          class="dp-button button-medium secondary-green button--round"
+        >
+          Back
+        </a>
+      </header>
+
+      <div class="dp-gray-page p-t-54 p-b-54">
+        <div class="dp-align-center">
+          <div>
+            <h3 class="m-l-30">Card Ribbon left-top</h3>
+            <div class="dp-card-ribbon-lt" data-ribbon="Recomendado"></div>
```

Comment:
What happens if you add more text here? Say like something that occupies two lines.

Reply:
![image](https://user-images.githubusercontent.com/2439363/88050161-b4c48900-cb2c-11ea-9d2e-e787a9e1283b.png)

This is to think for the future only, that´s all.

Reply:
http://cdn.fromdoppler.com/doppler-ui-library/build917/cards-component.html

Reply:
This does not work like that, there is no space for two words, the design is not prepared for this. It is not an elastic ribbon.
Surely if they want to add more words they will change the design.
This accepts 1 word or 2 very small

Reply:
https://cdn.fromdoppler.com/doppler-ui-library/build918/cards-component.html

Reply:
And in the case we need to support more than one size of ribbon, is it prepared? Or have you thought about this?

Reply:
In all the designs they have there are no variants of ribbons but it is something that can be done without difficulty 🐇

Reply:
Great job Gus! About the ribbons as Gus has said, there aren´t another iterations. Only it´s designed to contain one word, or two short words. If another requerimient shows up we will consider it! :)